src/systemctl/systemctl.c

```diff
@@ -241,7 +241,9 @@ static int systemctl_help(void) {
                "  -T --show-transaction  When enqueuing a unit job, show full transaction\n"
                "     --show-types        When showing sockets, explicitly show their type\n"
                "     --value             When showing properties, only print the value\n"
-               "  -i --ignore-inhibitors When shutting down or sleeping, ignore inhibitors\n"
+               "     --check-inhibitors=MODE\n"
+               "                         Specify if checking inhibitors before shutting down,\n"
+               "                         sleeping or hibernating\n"
```

Comment:
As mentioned I'd drop `--ignore-inhibitors` from the docs (and thus also the `--help` text), but keep `-i` as shortcut for `--check-inhibitors=no` in place.

The way I see short options is that they are shortcuts for certain things, i.e. their redundancy is inherent to their existance, and hence having redundant long options sucks and is something we should avoid/not document. But having short options that are partly redundant versions of long options is totally OK, it's why they exist.

i.e. add another line here:

```
" -i                     Shortcut for --check-inhibitors=no"
```

but don't mention --ignore-inhibitors anymore